GTG/gtk/browser/browser.py

```diff
@@ -1242,89 +1242,84 @@ def on_select_tag(self, widget=None, row=None, col=None):
 
         self.applied_tags = new_taglist
 
-    def on_taskdone_cursor_changed(self, selection=None):
-        """Called when selection changes in closed task view.
-
-        Changes the way the selected task is displayed.
+    def on_task_cursor_changed(self, selection):
+        """If there is an active task selected, unselect all the closed tasks
+        and set the status of the selected active task to STA_ACTIVE.
+        If no tasks at all are selected, set status to STA_ACTIVE"""
+
+        if selection.count_selected_rows() > 0 and
+        'closed' in self.vtree_panes:
```

Comment:
Can you indent it by 2x4 spaces to make sure that it is visually still part of the condition?